### ISSUE TITLE
Movie sets (potentially) not working

### ISSUE THREAD

**Issue by TheOddLinguist** [Jasonra/XBMC-PseudoTV#4]
I figured I'd add this here so you don't have to track down my post in the XBMC forum.

---

I have three playlists:
- Sitcoms (Channel_1) - limited to 10 episodes of a few selected shows
- Other Shows (Channel_2) - limited to 10 episodes a few different shows
- Some Movies (Channel_3) - a few movies

The Some Movies playlist works when the options are:

```
Match one or more of the following
- Title is A Christmas Story
- Title is Airplane
- Title is Ocean's Eleven
```

When I add "Title contains Back to the Future", this includes the three BTTF movies, which are in a "Back to the Future" set. With this rule included, the add-on hangs on the channel list building screen.

Here are some debug logs:

The former setup, which works: http://pastebin.com/w7HUAib4

The latter setup, which does not: http://pastebin.com/34vUEctG

Hope that helps...let me know if you need anything else.

**Comment by TheOddLinguist** [comment]
Just a note, I turned on "Force all channels to be reset" before each time.

**Comment by Jasonra** [comment]
Fixed the freeze, but the short-term fix is that movie sets aren't added to a channel.  Should properly open and add them.

**Comment by TheOddLinguist** [comment]
I've been trying like crazy to make them work in my fork, but I can't seem to get it.  hehe  I'll let you know if I come up with anything, though.

**Comment by TheOddLinguist** [comment]
parse list using JSON-RPC&#39;s GetDirectory; closed by 924fbad1e99340c684987371cfd5f23c683d98fc
